src/main/scala/com/aivean/royalroad/Main.scala

```diff
@@ -191,6 +191,17 @@ object Main extends App {
               println("removing warning: " + p.text)
               p.underlying.remove()
           }
+
+          // find all divs, since royal road put warnings in divs now as well
+          val warningDivs = chapterContent.select("div")
```

Comment:
I believe this duplication could be avoided by just changing the selector above to include both divs and paragraphs:

```
val paragraphs = chapterContent.select("p,div")
```

could you please try that?

Reply:
Yep, that is a much cleaner way of doing the same thing. Tested and it works. I updated the pull request.

Thanks for making this tool ;-).

Reply:
thanks for contributing!